pyomo/environ/__init__.py

```diff
@@ -9,9 +9,11 @@
 #  ___________________________________________________________________________
 
 import sys as _sys
+
```

Comment:
I don't know why this and all the changes to this file (except line 63) were introduced (I want to blame Pycharm), but let's stick to what master says in terms of this file. The only change that should appear is the addition of `'pyomo.contrib.community_detection',` to `_optional_packages`